config/config.go

```diff
@@ -178,6 +177,8 @@ func DefaultTestConfig() Config {
 	conf.P2P = p2p.DefaultConfig()
 	conf.API = grpcserver.DefaultTestConfig()
 	conf.POSTService = activation.DefaultTestPostServiceConfig()
+	conf.HARE3.PreroundDelay = 1 * time.Second
+	conf.HARE3.RoundDuration = 1 * time.Second
```

Comment:
The defaults appear to be 25 and 12 seconds and I assume those are currently used on mainnet. Is it safe to just change those in this PR?